Source/Editor/Windows/Assets/PrefabWindow.Hierarchy.cs

```diff
@@ -366,6 +366,7 @@ public void Spawn(Actor actor)
                 actor.Transform = parentActor.Transform;
                 actor.StaticFlags = parentActor.StaticFlags;
                 actor.Layer = parentActor.Layer;
+                actor.LocalTransform = Transform.Identity;
```

Comment:
Because you added this here, you can remove the transform setting above because there is no need to set the transform 2 times. Or you can just use this line of code to replace the other line where the transform is being set (line 366).

Reply:
Makes more sense to me to remove line 366 and replace it with what you wrote in line 369.

Reply:
But doesn't that line move the actor to where it's parent is?

Reply:
No, since you are setting the local transform, as long as the actor has a parent set, it will spawn it correctly.

Reply:
Okay I tried your suggestion and you are right!

Reply:
You can test it and see. It seemed to work for me when I tested it.

Reply:
Yes I made the change